assets/scss/frontend/_llms-notifications.scss

```
@@ -67,18 +67,45 @@
 
 				.llms-mini-cert {
 					background: #f6f6f6;
-					border: 4px double #b0b0b0;
-					padding: 24px 8px;
+					border: 1px solid #d0d0d0;
+					box-shadow: inset 0 0 0 3px #fefefe, inset 0 0 0 4px #dedede;
+					padding: 16px 16px 24px;
+					margin-top: 12px;
 					.llms-mini-cert-title {
 						font-size: 16px;
 						font-weight: 700;
-						margin-bottom: 8px;
+						margin-bottom: 12px;
```

```suggestion
						margin: 12px auto;
```
Let's set the margin top too. I was using 2020 theme and it had too much space above by default.